data/yam/agama/auto/default.jsonnet

```diff
@@ -0,0 +1,28 @@
+{
+  product: {
+    id: "{{PRODUCT_ID}}",
+    registrationCode: "{{SCC_REGCODE}}"
+  },
+  user: {
+    fullName: "Bernhard M. Wiedemann",
+    password: "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
+    hashedPassword: true,
+    userName: "bernhard"
```

Comment:
please, use single quotes for all the values not expected or not having spaces.

Reply:
updated.